### PR TITLE
Add Debian 12.7 among container test images

### PR DESCRIPTION
**Pull request by happz**
Pull Request Checklist

* [x] implement the feature

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -111,7 +111,8 @@ TMT_TEST_CONTAINER_IMAGES := $(TMT_TEST_CONTAINER_IMAGE_NAME_PREFIX)/alpine:late
                              $(TMT_TEST_CONTAINER_IMAGE_NAME_PREFIX)/fedora/39/upstream:latest \
                              $(TMT_TEST_CONTAINER_IMAGE_NAME_PREFIX)/fedora/39/unprivileged:latest \
                              $(TMT_TEST_CONTAINER_IMAGE_NAME_PREFIX)/ubi/8/upstream:latest \
-                             $(TMT_TEST_CONTAINER_IMAGE_NAME_PREFIX)/ubuntu/22.04/upstream:latest
+                             $(TMT_TEST_CONTAINER_IMAGE_NAME_PREFIX)/ubuntu/22.04/upstream:latest \
+							 $(TMT_TEST_CONTAINER_IMAGE_NAME_PREFIX)/debian/12.7/upstream:latest
```
Comment (thrix) [inline Makefile:115]: bad ident?

File: containers/debian/12.7/Containerfile.upstream [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#
+# A Debian image tailored for tmt test suite
+#
+# tmt/tests/debian/12.7/upstream:latest
+#
+
+FROM docker.io/library/debian:12.7
+
+    # Populate apt cache
```
Comment (thrix) [inline containers/debian/12.7/Containerfile.upstream:9]: weird ident

Comment (happz) [reply containers/debian/12.7/Containerfile.upstream:9]: This is expected, it's aligned with the command it describes, not with `RUN`.

File: Makefile (hunk 1 above)
Comment (happz) [reply Makefile]: Fixed in 0e7a5a6b

File: containers/debian/12.7/Containerfile.upstream (hunk 2 above)
Comment (psss) [reply containers/debian/12.7/Containerfile.upstream:9]: Interesting, seems weird to me as well. Ale proti gustu... ;-)

**Review by psss — APPROVED** [review]
Looks good, thanks!

File: containers/debian/12.7/Containerfile.upstream (hunk 2 above)
Comment (happz) [reply containers/debian/12.7/Containerfile.upstream:9]: It's unusual, and it's caused by that `RUN` command. It starts to be less distracting once there are more commands:

```dockerfile
    # Populate dnf cache
RUN    dnf makecache \
    # Create unprivileged user and setup sudo for it
    && dnf install -y /usr/sbin/useradd \
    && useradd fedora \
    && usermod -aG wheel fedora \
    && echo -e 'fedora\tALL=(ALL)\tNOPASSWD: ALL' >> /etc/sudoers
```

Maybe we could put `RUN` on its own line, and help the visual impression a bit by changing it to this?

```dockerfile
RUN \
    # Populate dnf cache
    && dnf makecache \
    # Create unprivileged user and setup sudo for it
    && dnf install -y /usr/sbin/useradd \
    && useradd fedora \
    && usermod -aG wheel fedora \
    && echo -e 'fedora\tALL=(ALL)\tNOPASSWD: ALL' >> /etc/sudoers
```

Comment (psss) [reply containers/debian/12.7/Containerfile.upstream:9]: No problemo, it's fine. We can improve if/when there are more lines.

**Comment by psss** [comment]
The failing test was a network glitch.

**Pull request merged**